apps/storybook/src/decorators/FillHeight.tsx

```diff
@@ -1,9 +1,14 @@
-import type { Story } from '@storybook/react';
+import type { StoryContext, StoryFn } from '@storybook/react';
 
-function FillHeight(MyStory: Story) {
+function FillHeight(Story: StoryFn, context: StoryContext) {
   return (
-    <div style={{ display: 'grid', height: '100vh' }}>
-      <MyStory />
+    <div
+      style={{
+        display: 'grid',
+        height: context.viewMode === 'story' ? '100vh' : '25rem', // fixed height in docs mode
```

Comment:
This improves documentation pages by reducing the height of the iframes of the documented stories.